src/routes/acquirerRoutes.ts

```diff
@@ -123,15 +134,40 @@ router.get("/:resourceID/:step", async (req: Request, res: Response) => {
   const stepData = formdata.steps[formStep];
   const assetTitle = formdata.assetTitle;
 
+  if (req.query.action === 'back' && formdata.stepHistory) {
+    formdata.stepHistory.pop();
+  }
+
   if (skipThisStep(formStep, formdata)) {
     stepData.skipped = true;
     return res.redirect(`/acquirer/${resourceID}/${stepData.nextStep}`);
   }
+  
+  let backLink = null; 
+
+  if (!formdata.stepHistory) {
+    formdata.stepHistory = [];
+  }
+  
+  // If we have history and are pressing back, we set the back link.
+  if (formdata.stepHistory.length > 1) {
+    backLink = `/acquirer/${resourceID}/${formdata.stepHistory[formdata.stepHistory.length - 2]}?action=back`;
+ }
+
+ if (formStep === 'data-type') {
+  // If current step is 'data-type', set the back link to start page -> 
+  // in preperation for Maddies current work before Annual leave data-type being the only page to start from
+  backLink = `/acquirer/${resourceID}/start`;
+} else if (formdata.stepHistory && formdata.stepHistory.length > 1) {
+  // Otherwise, set it to the previous step from stepHistory
+  backLink = `/acquirer/${resourceID}/${formdata.stepHistory[formdata.stepHistory.length - 2]}?action=back`;
+}
```

Comment:
I think there are a couple of numbers off-by-one here. This seems to be working for me:
```suggestion
  if (formdata.stepHistory.length > 0) {
    // Set the backlink to the previous step from stepHistory, if we've got some history
    backLink = `/acquirer/${resourceID}/${formdata.stepHistory[formdata.stepHistory.length - 1]}?action=back`;
  } else {
   // Otherwise set the backlink it to /start
    backLink = `/acquirer/${resourceID}/start`;
  }
```

Reply:
nice catch, still seems to be an issue if you go through data-type to project-aims and hit back you're taken to data-type, skips the data-subject page

Reply:
Nice catch, thank you Kiran. I have made the change 👍🏼